### ISSUE TITLE
Possible bug exposed by the planetarium app

### ISSUE THREAD

**Issue by liaxim** [Samsung/GearVRf#644]
While testing the picking I noticed some of the planets were missing textures (getting green for mercury for example). The logs are littered with this:

```
06-27 08:06:01.198  5347  5757 E gvrf    : Error detected in Renderer::renderRenderData; name : 4, error : Light::getVec4() : specular_intensity not found
06-27 08:06:01.198  5347  5757 E gvrf    : Error detected in Renderer::renderRenderData; name : 1, error : Light::getVec4() : specular_intensity not found
06-27 08:06:01.228  5347  5757 E gvrf    : Error detected in Renderer::renderRenderData; name : 9, error : Light::getVec4() : specular_intensity not found
```

**Comment by rahul27** [comment]
You need to add the following line:

GVRLight mLight = new GVRLight(gvrContext);
mLight.setAmbientIntensity(0.5f, 0.5f, 0.5f, 1.0f);
mLight.setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
**mLight.setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);**

We could set it as default for apps not using this call. @roshanch will know better.

**Comment by liaxim** [comment]
Sounds like a reasonable default for the compatibility GVRLight since its use in code based on an older fw code doesn't require the extra call. Btw, should GVRLight be deprecated?

**Comment by roshanch** [comment]
Yes, we need to add default values for the light components. About deprecation, not sure whether its a good idea. adding @NolaDonato

**Comment by NolaDonato** [comment]
Planetarium uses GVRLight which inherits from GVRPointLight. We need to set the default specular intensity in GVRPointLight. I can add this fix.

**Comment by NolaDonato** [comment]
Yes. It is really just GVRPointLight.
